### DISCUSSION TITLE
Collection of comments on User Guide

### DISCUSSION THREAD

**Discussion by vonglan** [helgoboss/helgobox#309] [Ideas]
Very low prio.

5.2.5.1 For all source characters:
- Shouldn't these be explained in the order the appear in the GUI, i.e. Source Min/max before Target Min/Max?
- The introductory sentence part "both in control and feedback direction" is not 100% correct, e.g. the Feedback transformation. Maybe just delete those 6 words, or add "most of them"
- Change "For rotary encoders" to "relative encoders"? Or should this be "relative control sources" because it also applies to Virtual Multis with relative content?
- Source Min/Max: I did not understand "In relative mode, this only has an effect on absolute source control values, not on relative ones". 
- Maybe "relative mode" and "absolute mode" are ReaLearn 1.0 terms? (They also appear in "Out-of-range-behavior" section). 
- In the rendering on the website, the table in "Out-Of-Range" is badly formatted. The github .md version looks fine.

5.2.5.2 For knobs/faders and buttons
- "only relevant for sources that emit absolute values - knobs, faders, buttons" - how about Virtual Multis? **Only when they carry absolute values as content?** 
- Mode / Toggle Buttons: "toggle a target between on and off states" add "(or 2 other states)". I used this (Toggle between Unison and Poly1 Mode, where 4 different modes are available)
- Mode / Toggle Buttons: in the same position, also add: "each time a value other than 0 is received".
- Mode / Toggle Buttons: "only makes sense for momentary buttons" - add: "or sources with character "toggle-only""

5.2.5.3 For encoders and incremental buttons
- Speed min/max. I would move the sentence "Virtual targets are always discrete" into the beginning sentence "a discrete target (or a virtual one)". Similarly, under "step size", you could say "continuous targets (but not virtual ones)". Or add "but always/never in a Controller Mapping"
- Maybe start with "Speed" (because this is in the Controller Mapping), instead of "Step size"?
- Are FX parameters always "continuous"?

**Comment by helgoboss** [comment]
Really cool that you invest so much thought into that. I will have a closer look soon.

**Comment by vonglan** [comment]
Yes, I can get quite detailed.  Which in some contexts can be annoying. And in other contexts it's very useful.

By the way, I marked one question in **bold face**, which seems important to me.

**Comment by helgoboss** [comment]
Hi @vonglan. With the new [documentation website](https://docs.helgoboss.org/realearn) in place It's now possible to suggest changes to the docs by pressing "Edit this page". See announcement: https://forum.cockos.com/showthread.php?t=295379

**Comment by vonglan** [comment]
Hi Benjamin, thanks.
